src/status_im/ui/market/view.cljs

```diff
@@ -0,0 +1,98 @@
+(ns status-im.ui.market.view
+  (:require
+    [quo.context]
+    [quo.core :as quo]
+    [quo.foundations.colors :as colors]
+    [react-native.core :as rn]
+    [status-im.common.home.top-nav.view :as common.top-nav]
+    [status-im.common.refreshable-flat-list.view :as refreshable-flat-list]
+    [status-im.ui.market.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn swap-header
+  []
+  (let [theme             (quo.context/use-theme)
+        operable-accounts (rf/sub [:wallet/operable-accounts])]
+    [rn/view {:style (style/swap-header-container theme)}
+     [quo/text
+      {:weight :semi-bold
+       :size   :heading-1
+       :style  (style/market-header-text theme)}
+      (i18n/label :t/market)]
+     [quo/button
+      {:size      32
+       :icon-left :i/swap
+       :on-press  #(rf/dispatch [:app.wallet/start-swap operable-accounts])}
+      (i18n/label :t/swap)]]))
+
+(defn sort-header
+  []
+  (let [theme (quo.context/use-theme)]
+    [rn/view {:style (style/sort-header-container theme)}
+     [quo/text
+      {:size  :paragraph-2
+       :style (style/sort-text theme)}
+      (i18n/label :t/market-cap)]
+     [quo/icon :i/arrow-down
+      {:size            12
+       :color           colors/neutral-50
+       :container-style {:margin-left 4}}]]))
+
+(defn token
+  [token-id]
+  (let [{:keys [price percentage-change market-cap short-name token-rank token-name image]}
+        (rf/sub [:ui.market/token token-id])]
+    [quo/market-token
+     {:token-short-name  short-name
+      :token-rank        token-rank
+      :token-name        token-name
+      :market-cap        market-cap
+      :price             price
+      :token-image       image
+      :percentage-change percentage-change}]))
+
+(defn ensure-tokens-updating
+  [event]
+  (let
+    [visible-tokens (-> event
+                        .-viewableItems
+                        js->clj)
+     token-at-top   (-> visible-tokens
```

Comment:
If you only need to convert to CLJS the first item it's better not to convert the entire `viewableItems` into CLJS.